packages/esm-patient-chart-app/translations/fr.json

```diff
@@ -34,7 +34,6 @@
   "diagnoses": "Diagnostics",
   "discard": "Abandonner",
   "dose": "Dose",
-  "edit": "Editer",
```

Comment:
What is the reason for updating the transitions? @RandilaP

Reply:
It happened automatically when I commit my changes